cli/src/utils/getdeps.ts

```diff
@@ -38,9 +38,23 @@ export async function getDependencies(projectDir: string) {
     if (p === '@reshuffle/code-transform') continue;
     // exclude already visited deps
     if (dependencies.has(p)) continue;
-    dependencies.add(p);
-    for (const subpackage of Object.keys(lockDeps[p].requires || {})) {
-      toProcess.push(subpackage);
+
+    const lockDep = lockDeps[p];
+    if (lockDep) {
+      dependencies.add(p);
+      if (has(root, 'dependencies')) {
```

Comment:
Should it be `if (has(lockDep, 'dependencies'))` since root will never have dependencies?